session/manager.go

```diff
@@ -84,12 +86,12 @@ type Manager interface {
 	// UpsertAndIssueCookie stores a session in the database and issues a cookie by calling IssueCookie.
 	//
 	// Also regenerates CSRF tokens due to assumed principal change.
-	UpsertAndIssueCookie(context.Context, http.ResponseWriter, *http.Request, *Session) error
+	UpsertAndIssueCookie(context.Context, http.ResponseWriter, *http.Request, *Session, ...x.CookieModifier) error
 
 	// IssueCookie issues a cookie for the given session.
 	//
 	// Also regenerates CSRF tokens due to assumed principal change.
-	IssueCookie(context.Context, http.ResponseWriter, *http.Request, *Session) error
+	IssueCookie(context.Context, http.ResponseWriter, *http.Request, *Session, ...x.CookieModifier) error
```

Comment:
Please check all places where these two functions are being used and interpret whether they issue a new cookie with a new nonce or not.

Reply:
3 usages of Upsert and Issue -> all perform prior activation of session (followed by addition of nonce) which results in a new kratos session (Login, Code and Link recovery).
3 usages of Issue -> nonce addition to the values map results in a new kratos session (PostSettingsHook, Post Registration, Refresh cookie)